app/assets/v2/js/pages/dashboard.js

```diff
@@ -370,9 +370,14 @@ var get_search_URI = function(offset, order) {
     order_by = localStorage['order_by'];
   }
 
-  if (typeof order_by !== 'undefined') {
-    uri += '&order_by=' + order_by;
+  if (!document.hackathon) {
```

Comment:
No bounties are showing up for me in the explorer

Reply:
was filtering right ? because is ok in my side